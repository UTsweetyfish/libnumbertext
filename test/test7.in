ordinal 2
ordinal 1002
ordinal
2
