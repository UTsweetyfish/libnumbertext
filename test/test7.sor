# check prefix macro

1 one

== ordinal ==

1 first
^2 secondA
2 secondB
3 third
(\d)(\d{3}) $1 thousand $(ordinal \2)
"" Ordinal example...

== ==

2 two
