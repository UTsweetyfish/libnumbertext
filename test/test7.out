secondA
one thousand secondB
Ordinal example...
two
